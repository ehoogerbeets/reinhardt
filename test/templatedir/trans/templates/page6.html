{% trans "This is a trans tag with a source string and a translators comment" comment "This is also a translator's comment" %}
