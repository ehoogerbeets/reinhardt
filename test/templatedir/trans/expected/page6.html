Dies ist eine trans-Tag mit einem Ausgangssatz und einem Ãœbersetzer Kommentar
